### DISCUSSION TITLE
there’s a lot of miner606 channels

### DISCUSSION THREAD

**Discussion by ayjemabo** [ThioJoe/YT-Spam-Lists#199] [Ideas]
![image](https://user-images.githubusercontent.com/95683302/193443315-aae0c9d4-00cd-4662-9f5d-964e08bf3aab.jpeg)
look at the channels
it's all ```
Message Miner606 on telegram
```

should we add the message channels?

**Comment by Firecul** [comment]
I've added that one to the lists with a75d56901b6cc344205afaac948e90ce6f01ed8b
